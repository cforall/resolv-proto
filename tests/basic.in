1 f 2 3
2 g 2
%%
f( g(2) 3 )
