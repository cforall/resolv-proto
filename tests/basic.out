[1 / (0,0,0,0,0)] f( g( 2 ) 3 )
